mobile.html

```diff
@@ -29,13 +29,13 @@
                 <div class="group-4">
                     <div class="group">
                         <div class="cellular">
-                            <img src="icons/CellularConnection.png" alt="">
+                            <img src="icons/CellularConnection.png" alt="cellular icon">
                         </div>
                         <div class="wifi">
-                            <img src="icons/Wifi.png" alt="">
+                            <img src="icons/Wifi.png" alt="wifi-icon">
                         </div>
                         <div class="battery">
-                            <img src="icons/Battery.png" alt="">
+                            <img src="icons/Battery.png" alt="battery-icon">
                         </div>
                     </div>
                    
```

Comment:
- Kindly provide the requested design(font-family and align text at the center):
![Screen Shot 2021-09-28 at 9 55 52](https://user-images.githubusercontent.com/22258274/135112547-1b2c79fb-c608-4633-8041-502f6369f6b8.png)

Reply:
> * Kindly provide the requested design(font-family and align text at the center):
>   ![Screen Shot 2021-09-28 at 9 55 52](https://user-images.githubusercontent.com/22258274/135112547-1b2c79fb-c608-4633-8041-502f6369f6b8.png)

Hey Carlos, please can you throw more light on this requested change.